bitnami/elasticsearch/templates/curator/cronjob.yaml

```diff
@@ -8,55 +8,50 @@ metadata:
     ## Istio Labels: https://istio.io/docs/ops/deployment/requirements/
```

Comment:
Putting it here because I can't put it above

The helper `cronjob.apiVersion` should be changed by `common.capabilities.cronjob.apiVersion`